### PR TITLE
Clarifying MT + Identity

### PR DESCRIPTION
**Pull request by akeller**
## Description

Related to this internal thread - https://camunda.slack.com/archives/CRQAL5A1M/p1702594236648519

## When should this change go live?

- [ ] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [x] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [x] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [x] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
Question for my panel of amazing reviewers - if authorization through Identity must be enabled for multi-tenancy, should we add another bullet point at the top of the **Configure multi-tenancy** guide?

Are there other areas we need to clarify around authorization?

**Comment by akeller** [comment]
I blew up the scope a bit from where I started, but it was necessary. This PR adds an operational guide for setting up MT. We'll follow this up in Q1 with an E2E user guide for MT via https://github.com/camunda/product-hub/issues/1960.

This is my gift to you for the end of the year. 🎁 I'm headed on vacation until Jan 2, so you have some options if you'd like to get this ready, reviewed, and merged ASAP. @christinaausley, I nominate you as the docs coordinator for this PR.

If you'd like to address [my comment here](https://github.com/camunda/camunda-docs/pull/3076#issuecomment-1858501461) with some consensus among the reviewers, go for it. Otherwise, I can follow up if there is still confusion. 

Also, @christinaausley consider https://github.com/camunda/camunda-docs/pull/3048 and whether we want to link to any of the individual component troubleshooting sections you are working on with that PR. We love a good cross link! 

If there are significant concerns or issues with this PR and you think it would be better to leave the MT + Identity docs as-is, that's fine too. It would be great to collect the feedback here in this PR so I can address it on my return.

File: docs/self-managed/concepts/multi-tenancy.md [hunk 1]
```diff
@@ -9,26 +9,13 @@ description: "Multi-tenancy allows you to re-use your Camunda installation."
...
-
-- [Identity](../../../self-managed/identity/deployment/configuration-variables/#feature-flags)
-- [Zeebe](../../../self-managed/zeebe-deployment/configuration/gateway-config/#zeebegatewaymultitenancy)
-- [Operate](../../../self-managed/operate-deployment/operate-configuration/#multi-tenancy)
-- [Tasklist](../../../self-managed/tasklist-deployment/tasklist-configuration/#multi-tenancy)
-- [Optimize]($optimize$/self-managed/optimize-deployment/configuration/multi-tenancy/)
-- [Connectors](../../../self-managed/connectors-deployment/connectors-configuration/#multi-tenancy)
-
-Unexpected behavior may occur if multi-tenancy is only enabled in some components.
+See the [configuration guide](/self-managed/operational-guides/configure-multi-tenancy.md) for details on how to enable multi-tenancy.
```
Comment (koevskinikola) [inline docs/self-managed/concepts/multi-tenancy.md:12]: :+1: Nice!

**Review by koevskinikola — DISMISSED** [review]
LGTM! :rocket:

**Comment by christinaausley** [comment]
Looping in @Ben-Sheppard and @nloding -- any final thoughts or feedback here?

**Comment by akeller** [comment]
I cleaned up the merge conflicts and this is ready for a final review.

**Review by christinaausley — DISMISSED** [review]

**Comment by akeller** [comment]
Merge conflicts again 😅 they don't stop! I will clean this up after my meeting and enable auto-merge.

**Review by christinaausley — APPROVED** [review]
Done @akeller ! 🥳

**Pull request merged**